### ISSUE TITLE
[Core Mechanic]Windroc Matriarch#19055 unable to attack pets & Pets not running towards it either, when set to attack

### ISSUE THREAD

**Issue by Tenoutoften** [Atlantiss/NetherwingBugtracker#2106]
**Description**: 
[[NPC]Windroc Matriarch#19055](https://www.wowhead.com/npc=19055/windroc-matriarch) is unable to attack pets(seemingly). This [video](https://www.youtube.com/watch?v=Ve9Wp0EcD-0&feature=youtu.be) also features another issue with the pet not wanting to run to the mob and attack either, instead it stands still as if it was a totem. Pets seem to not want to run towards other bird mobs, and they seem to not want to attack pets:

[[NPC]Kaliri Matriarch#17035](https://db.excalibur.ws/?npc=17035)
[[NPC]Kaliri Swooper#17053](https://db.excalibur.ws/?npc=17053)
[[NPC]Female Kaliri Hatchling#17034](https://db.excalibur.ws/?npc=17034)
[[NPC]Grishna Raven#20327](https://db.excalibur.ws/?npc=20327)

Basically mobs that seem like they can fly, not limited to birds. Apparently happens with Chimeras & faerie dragons in BEM too(Don't got precise names or IDs)

**Expected behaviour**: 
Expected to be able to hit pets / pets suppose to be able to run towards them and attack them as regular mobs.

**Server Revision**: 
2244

**Comment by xmesaj2** [comment]
#2299
